app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
```

Comment:
**REQUIRED**
* username is validated (at least for the presence) (covered by automated tests) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/userValidation.png) from Odin Project)